timemachine/lib/potentials.py

```diff
@@ -1,13 +1,17 @@
+import pathlib
 from typing import List
 
 import numpy as np
 from numpy.typing import NDArray
 
+import timemachine
 from timemachine.lib import custom_ops
 
 # (ytz): classes in this class wrap custom_ops but have the added benefit
 # of being pickleable.
 
+_kernel_source_dir = str(pathlib.Path(timemachine.__file__).parent / "cpp" / "src" / "kernels")
```

Comment:
Oh, that's a good catch. Should probably be consistent and use `importlib` here too.

Reply:
On the other hand, importlib would require cpp to be a python package? That's going to look odd...

Reply:
Here's what it'd look like with `importlib`. Unfortunately `importlib` in Python 3.7 doesn't include the `files` helper, and the recommendation seems to be to pull in the backported `importlib_resources` package.

> On the other hand, importlib would require cpp to be a python package? That's going to look odd...

Agreed about this; however I'm currently leaning toward this method given that it appears to be the recommended approach, and the feature of being format-agnostic seems like it might prevent headaches down the road if we ever choose to distribute timemachine on PyPI, for example. What do you think?

Reply:
Yeah, I agree, importlib seems to be the way to go. Maybe add a comment in the the cpp __init__ (this file is present for JIT compilation of certain kernels)

Reply:
Good call, added in 6a37006